packages/suite/src/actions/suite/desktopUpdateActions.ts

```diff
@@ -46,20 +48,42 @@ export const downloading = (progress: UpdateProgress): DesktopUpdateAction => ({
     payload: progress,
 });
 
-export const ready = (info: UpdateInfo): DesktopUpdateAction => ({
-    type: DESKTOP_UPDATE.READY,
-    payload: info,
-});
+export const ready = (info: UpdateInfo) => (dispatch: Dispatch, getState: GetState) => {
+    const { latest } = getState().desktopUpdate;
+
+    // update can fail even if it was downloaded successfully
+    // however, it should affect only a small portion of users
```

Comment:
What about to use `update/new-version-first-run`/`newVersionFirstRun` and report from there? 

I mean renaming this status to something like `ready`, passing also `savedCurrentVersion` in `update/new-version-first-run` and reporting from `newVersionFirstRun` the finished status (or something new like `new-version`)?

Reply:
Sounds like a good idea. I also have to save and pass `prerelease`

Reply:
release vs pre-release state is dynamic, could change at any time just by click on github... Maybe the only important info to check is whether the user is in Early Access Program or not?

Reply:
Or we can track both

Reply:
@matejkriz Please check fixup. We can add `newVersionFirstRun` status later when we add analytics to the electron layer.

- [x] notion updated

Reply:
looks great, let's squash and merge it 👍